x-pack/test/cloud_security_posture_functional/cloud_tests/basic_ui_sanity.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const retry = getService('retry');
+  const pageObjects = getPageObjects(['common', 'cloudPostureDashboard', 'header']);
+
+  describe('Cloud Posture Dashboard Page', function () {
+    this.tags(['cloud_security_posture_ui_sanity']);
+    let cspDashboard: typeof pageObjects.cloudPostureDashboard;
+    let dashboard: typeof pageObjects.cloudPostureDashboard.dashboard;
+
+    before(async () => {
+      cspDashboard = pageObjects.cloudPostureDashboard;
+      dashboard = pageObjects.cloudPostureDashboard.dashboard;
+      await cspDashboard.waitForPluginInitialized();
+      await cspDashboard.navigateToComplianceDashboardPage();
+      await retry.waitFor(
+        'Cloud posture integration dashboard to be displayed',
+        async () => !!dashboard.getIntegrationDashboardContainer()
+      );
+    });
+
+    describe('Cloud Dashboard', () => {
+      it('displays accurate summary compliance score', async () => {
+        await pageObjects.header.waitUntilLoadingHasFinished();
+        const scoreElement = await dashboard.getCloudComplianceScore();
+        expect((await scoreElement.getVisibleText()) === '41%').to.be(true);
+      });
+
+      it('displays all compliance scores', async () => {
+        const scoresElements = await dashboard.getAllCloudComplianceScores();
+        const scores: string[] = [];
+        for (const scoreElement of scoresElements) {
+          scores.push(await scoreElement.getVisibleText());
+        }
+        // 3 scores for each cloud provider + 1 summary score
+        expect(scores.length).to.be(4);
+      });
+
+      it('displays a number of resources evaluated', async () => {
+        const resourcesEvaluated = await dashboard.getCloudResourcesEvaluated();
+        const visibleText = await resourcesEvaluated.getVisibleText();
+        const resourcesEvaluatedCount = parseInt(visibleText.replace(/,/g, ''), 10);
+        expect(resourcesEvaluatedCount).greaterThan(3000);
+      });
+    });
+
+    describe('Kubernetes Dashboard', () => {
+      it('displays accurate summary compliance score', async () => {
+        await pageObjects.header.waitUntilLoadingHasFinished();
+        const scoreElement = await dashboard.getKubernetesComplianceScore();
+        expect((await scoreElement.getVisibleText()) === '83%').to.be(true);
```

Comment:
More like the previous question, will the agents be installed pointing to a controlled environment that the score doesn't (or at least shouldn't) change over time? otherwise, we could perform an assertion that are more flexible:

```js
expect(score).greaterThan(80);
```

The main reason is having to avoid having to update the tests in Kibana whenever these values change if the findings of the environments scanned are mutable

Reply:
fixed